### PR TITLE
Open/Controlled facet updates

### PR DESCRIPTION
**Pull request by rushtong**
Fix for https://broadinstitute.atlassian.net/browse/GAWB-2603

---

- [x] **Submitter**: Include the JIRA issue number in the PR description
- [x] **Submitter**: Check documentation and code comments. Add explanatory PR comments if helpful.
- [x] **Submitter**: If you changed a URL that is used elsewhere (e.g. in an email), comment about where it is used and ensure the dependent code is updated.
- [x] **Submitter**: JIRA ticket checks:
  * Acceptance criteria exists and is met
  * Note any changes to implementation from the description
  * To Demo flag is set
  * Release Summary is filled out, if applicable
  * Add notes on how to QA
- [x] **Submitter**: Update RC_XXX release ticket with any config or environment changes necessary
- [x] **Submitter**: Update FISMA documentation if changes to:
  * Authentication
  * Authorization
  * Encryption
  * Audit trails
- [x] **Submitter**: If you're adding new libraries, sign us up to security updates for them
* Review cycle:
  * LR reviews
  * Rest of team may comment on PR at will
  * **LR assigns to submitter** for feedback fixes
  * Submitter rebases to develop again if necessary
  * Submitter makes further commits. DO NOT SQUASH
  * Submitter updates documentation as needed
  * Submitter **reassigns to LR** for further feedback
- [x] **TL** sign off
- [x] **LR** sign off
- [x] **Product Owner** sign off
- [ ] **Submitter**: Verify all tests go green, including CI tests
- [x] **Submitter**: Squash commits and merge to develop
- [x] **Submitter**: Delete branch after merge
- [x] **Submitter**: **Test this change works on dev environment after deployment**. YOU own getting it fixed if dev isn't working for ANY reason!
- [x] **Submitter**: Mark JIRA issue as resolved once this checklist is completed

### CODE REVIEW

**Review by davidangb — COMMENTED** [review]
works for me. I realize this is a little scope creep: could you also change "protected" to "controlled access" in library_page.cljs and profile.cljs?

**Comment by rushtong** [comment]
Updated!

File: src/cljs/main/broadfcui/page/workspaces_list.cljs [hunk 1]
```diff
@@ -158,10 +158,10 @@
     :predicate (fn [ws option] (= option (boolean (get-in ws [:workspace :attributes :library:published]))))}
    {:title "TCGA Access"
     :options [:open :protected]
-    :render #(if (= % :open) "TCGA Open Access" "TCGA Protected Access")
+    :render #(if (= % :open) "TCGA Open Access" "TCGA Controlled Access")
     :predicate (fn [ws option]
                  (and (= (config/tcga-namespace) (get-in ws [:workspace :namespace]))
-                      ((if (= option :open) not identity) (get-in ws [:workspace :authorizationDomain]))))}])
+                      ((if (= option :open) empty? not-empty) (get-in ws [:workspace :authorizationDomain]))))}])
```
Comment (MattPutnam) [inline src/cljs/main/broadfcui/page/workspaces_list.cljs]: `seq` instead of `not-empty`

**Review by MattPutnam — APPROVED** [review]
👍

**Review by davidangb — APPROVED** [review]
fwiw, I liked the symmetry of `empty? not-empty`, I think it's (slightly) easier to read than `empty? seq`. But I also don't care :) 👍

**Pull request merged**